pkg/controller/operator/defaults/defaults.go

```diff
@@ -410,6 +436,8 @@ func DefaultConfiguration(config *kubermaticv1.KubermaticConfiguration, logger *
 		return configCopy, err
 	}
 
+	configCopy.Spec.Versions.ExternalClusters = ExternalClusterDefaultKubernetesVersioning
+
```

Comment:
Please take a look at how defaultVersioning` only overwrites smaller things, not the entire version configuration. Something similar should happen here for EKS/AKS as well.

Reply:
done